tests/unit/test_pathutil.py

```diff
@@ -212,10 +212,13 @@ def test_make_suite_run_tree(
                 'share': '$DEE/cylc-run/suite3/share'})
     ], ids=["1", "2", "3"])
 def test_get_dirs_to_symlink(
-        suite, install_target, mocked_glbl_cfg, output, mock_glbl_cfg):
+        suite, install_target, mocked_glbl_cfg,
+        output, mock_glbl_cfg, tmp_path):
+    os.environ['DEE'] = f"{tmp_path}"  # Ensures dirs returned are unexpanded
     mock_glbl_cfg('cylc.flow.pathutil.glbl_cfg', mocked_glbl_cfg)
     dirs = get_dirs_to_symlink(install_target, suite)
     assert dirs == output
+    os.environ.pop('DEE')
```

Comment:
If you use `monkeypatch.setenv('DEE', "whatever")`, then there's no need to remove the env var at the end